### DISCUSSION TITLE
Any way to address `::-webkit-scrollbar-thumb:hover` pseudo?

### DISCUSSION THREAD

**Discussion by bram2000** [vanilla-extract-css/vanilla-extract#612] [Q&A] [answered]
As subject; does anyone know if there is anyway to access the `:hover` properties of the `::-webkit-scrollbar-thumb`?

**Comment by hamlim** [comment, marked as answer]
Can you share a snippet that isn't working? Or a codesandbox?

I'm seeing the styles get generated correctly using something like this:

```ts
export let scrollbarHover = style({
  selectors: {
    "&::-webkit-scrollbar-thumb:hover": {
      backgroundColor: "red"
    }
  }
});
```

**Reply by bram2000** [reply]
Thanks! That's exactly what I was looking for! I'm setting the other properties directly rather than using the `selectors`;

```javascript
    "::-webkit-scrollbar": {
        width: "15px",
    },
    "::-webkit-scrollbar-track": {
        borderRadius: "5px",
        background: "red",
    },
    "::-webkit-scrollbar-thumb": {
        borderRadius: "5px",
        background: "pink",
    },

```
